query/common/src/main/java/org/codice/ddf/admin/common/fields/base/BaseFunctionField.java

```diff
@@ -129,4 +165,10 @@ protected BaseFunctionField addMessages(Report report) {
         return addArgumentMessages(report.argumentMessages()).
                 addResultMessages(report.resultMessages());
     }
+
+    private String formatErrorCodes(Set<String> errorCodes) {
+        List sortedErrors = new ArrayList<>(errorCodes);
+        Collections.sort(sortedErrors);
+        return String.join(",    \r\n", sortedErrors);
```

Comment:
It's to do new line in markdown for graphQL to display it nicely. So it will do comma the new line after each error code.